packages/plugins/tanstack-query/src/runtime-v5/vue.ts

```
@@ -31,6 +35,17 @@ export { APIContext as RequestHandlerContext } from '../runtime/common';
 
 export const VueQueryContextKey = 'zenstack-vue-query-context';
 
+// from "@tanstack/vue-query"
+export type MaybeRefDeep<T> = MaybeRef<
+    T extends Function
```

Biome

[error] Don't use 'Function' as a type.

Prefer explicitly define the function shape. This type accepts any function-like value, which can be a common source of bugs.

(lint/complexity/noBannedTypes)